source/common/types/rewards-csv-request.types.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+export type GenerateRewardsCsvParams = {
+  rewards: Array<Array<string>>,
```

Comment:
Please improve this type. Export Reward as Array<string> and put here as Array<Rewards> or something similar. Also, apply that changes to all places where `Array<Array<string>>` exist.